python/TestHarness/schedulers/QueueManager.py

```diff
@@ -22,10 +22,10 @@ class QueueManager(Scheduler):
     pool once per group (see augmentJobs).
 
     Using this one unmodified job, the spec file involved is noted, and instructs the derived
-    scheduler how to launch this one single spec file (using --spec-file), along with any
+    scheduler how to launch this one single spec file (using --run-dir), along with any
```

Comment:
So I was unaware of where we were using --spec-file. Apparently it's used in the PBS launcher right here and we are relying on it to run a single file. Two confounding issues:
- We aren't testing PBS, which is admittedly very bad, but if we were this would hopefully fail, BUT...
- We have a logic error where we don't actually look at the argument for `--run-dir` without `-i` so we are missing 100% of the use case here and it wouldn't fail even if we ran it!